src/app/grants/page.tsx

```diff
@@ -23,6 +29,27 @@ const grantsPageBaseData = generateWebPageStructuredData({
   path: PATHS.GRANTS.path,
 })
 
+const grantsAndOpportunitiesData = [
+  {
+    title: 'Open Grants',
+    description:
+      'Open Grants support projects that improve Filecoin functionality and enhance network utility and directly contribute to the Foundation’s mission.',
+    Icon: OpenGrantsIconComponent,
+  },
+  {
+    title: 'Microgrant',
+    description:
+      'Documentation enhancement microgrants are available to support the improvement of existing documentation or the creation of new resources related to the Filecoin network.',
+    Icon: MicrograntsIconComponent,
+  },
+  {
+    title: 'Requests for Proposal',
+    description:
+      'Request for Proposals (RFP) grants have clearly scoped deliverables, milestones, and funding limits. Refer to GitHub for any ongoing RFPs.',
+    Icon: RequestForProposalIconComponent,
+  },
+]
+
```

Comment:
Happy to move this into `/_data` folder within` /grants` as per your own suggestion for the About page.